examples/cifar10/minimal_supervised_cifar10.py

```diff
@@ -23,7 +23,10 @@
 for i in range(60000):
     # Choose an output neuron to clamp to spiking behavior.
     c = choice(10, size=1, replace=False)
-    clamp = {'Ae': 10 * labels[i].long() + Tensor(c).long()}
+    clamp = torch.zeros(pipeline.time, network.n_neurons, dtype=torch.uint8)
+    clamp[c, :] = 1
+    clamp_v = torch.zeros(pipeline.time, network.n_neurons, dtype=torch.float)
+    clamp_v[c, :] = 10 * labels[i].float() + Tensor(c).float()
```

Comment:
This should be `clamp_v[10 * labels[i].long() + Tensor(c).long(), :] = -52`

-52 is the threshold value.

Reply:
10 * labels[i].long() + Tensor(c).long() > then index of clamp_v

Reply:
What? `labels[i]` is in `range(10)`, and `c` is in `range(10)`, so `10 * labels[i].long() + Tensor(c).long()` is in `range(100)`, which is the number of neurons in the example.

Reply:
Ok, I see what you mean. You mixed up between time axis and neuron axis... its the opposite from what you wrote.
I'm pushing the updated code